cloud/user-management.md

```diff
@@ -0,0 +1,71 @@
+# User management
+You can manage users for your Timescale Cloud account by navigating to the
+`Members` section in your Cloud console. From here, you can see the current
+members of your Timescale Cloud project, and add and remove members.
+
+## About Timescale Cloud user management
+Timescale Cloud allows you to collaborate with other users on your projects.
+When you have created your project, you can add other users so that they can see
+your project in their Timescale Cloud console. You can add, manage, and delete
+users from the `Members` section of the Cloud console.
+
+You are the assigned an `Administrator` role for projects that you create,
```

Comment:
That reads weird to my foreign eyes :grin:

Reply:
```suggestion
You are assigned an `Administrator` role for projects that you create,
```

Reply:
That's because it's a typo 😉